src/nxt_http_request.c

```diff
@@ -549,6 +549,34 @@ nxt_http_request_ready(nxt_task_t *task, void *obj, void *data)
     r = obj;
     action = r->conf->socket_conf->action;
 
+    if (r->chunked) {
+        r->transfer_encoding->skip = 1;
+        size_t size = r->body->file_end;
+
+        nxt_http_field_t *f;
+        f = nxt_list_zero_add(r->fields);
+        if (nxt_slow_path(f == NULL)) {
+            /* error */
+            return;
+        }
+
+        nxt_http_field_name_set(f, "Content-Length");
+
+        u_char *p, *end;
+        p = nxt_mp_nget(r->mem_pool, NXT_OFF_T_LEN);
+        if (nxt_slow_path(p == NULL)) {
+            /* error */
+            return;
+        }
+
+        f->value = p;
+        end = nxt_sprintf(p, p + NXT_OFF_T_LEN, "%O", size);
```

Comment:
Buildbot found following error when trying to compile with clang-ast on Debian 11:

```
src/nxt_http_request.c:573:55: error: bad argument for %O arg: size_t, resolved as: unsigned long
        end = nxt_sprintf(p, p + NXT_OFF_T_LEN, "%O", size);
```

Reply:
Forgot about that thing. We should look at moving that over to GH actions...

`size` is  a `size_t`

so

```
 *    %[0][width][u][x|X]z      ssize_t/size_t                                  
```

I guess it should be `%uz` instead...

Reply:
Fixed, thanks.

Reply:
Fixed, thanks.